scripts/processhit/processhit.gml

```diff
@@ -2,60 +2,119 @@
 // https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
 function ProcessHit( attackProperty, collision_list)
 {
-	// Combo Scaling
-	owner.combo++; // Add 1 to our combo length
-	var scaledDamage = attackProperty.damage[hitboxID]; // Set the initial amount of damage to do
-	var scaleAmount = 1 - (.1 * owner.comboScaling) // The amount to scale the combo by (decreases by 10% each for each scale)
-	scaleAmount = max(scaleAmount, ScalingMinimum);
-					
-	if (owner.combo > 2) 
-	{
-		scaledDamage *= scaleAmount; // The amount of damage this hit will do. Important that this is updated before scaling is updated
-		scaledDamage = round(scaledDamage); // Round the damage to the nearest whole number
-		scaledDamage = max(scaledDamage, 1); // The lowest amount of damage a move can do must be 1 HP
-		owner.comboScaling += attackProperty.comboScaling[hitboxID];
-	} // increase the level of scaling for the combo
-	if (owner.combo == 2)
+	if (!isProjectile)
 	{
-		owner.startCombo = true;
-	} // Tells the game to display the combo counter when the combo is at least 2 hits long
+		// Combo Scaling
+		owner.combo++; // Add 1 to our combo length
+		var scaledDamage = attackProperty.damage[hitboxID]; // Set the initial amount of damage to do
+		var scaleAmount = 1 - (.1 * owner.comboScaling) // The amount to scale the combo by (decreases by 10% each for each scale)
+		scaleAmount = max(scaleAmount, ScalingMinimum);
+					
+		if (owner.combo > 2) 
+		{
+			scaledDamage *= scaleAmount; // The amount of damage this hit will do. Important that this is updated before scaling is updated
+			scaledDamage = round(scaledDamage); // Round the damage to the nearest whole number
+			scaledDamage = max(scaledDamage, 1); // The lowest amount of damage a move can do must be 1 HP
+			owner.comboScaling += attackProperty.comboScaling[hitboxID];
+		} // increase the level of scaling for the combo
+		if (owner.combo == 2)
```

Comment:
This statement should be an else if since it can never execute unless the previous if fails.